pkg/kncloudevents/utils.go

```diff
@@ -26,6 +26,32 @@ import (
 	"github.com/cloudevents/sdk-go/v2/types"
 )
 
+const (
+	// AcceptReplyHeaderKey is the key for the event reply header.
+	// https://github.com/knative/eventing/blob/master/docs/spec/data-plane.md#event-reply-contract
+	AcceptReplyHeaderKey = "Prefer"
+	// AcceptReply is the value to accept event reply.
+	AcceptReply = "reply"
+)
+
+// SetAcceptReplyHeader sets Prefer: reply.
+func SetAcceptReplyHeader(headers nethttp.Header) {
+	if _, ok := headers[AcceptReplyHeaderKey]; !ok {
+		headers.Set(AcceptReplyHeaderKey, AcceptReply)
```

Comment:
Performance protip: the header name is already canonalized, you can just write the map

```
var acceptReplyHeaders = []string{AcceptReply} // outside the function, so you use always the same alloc
[...]
headers[AcceptReplyHeaderKey] = acceptReplyHeaders
```

Reply:
Make perfect sense. I like the reusing same alloc style. Thanks.